src/celeritas/field/CylMapFieldInput.hh

```diff
@@ -34,12 +34,12 @@ namespace celeritas
  */
 struct CylMapFieldInput
 {
-    std::vector<real_type> grid_r;  //!< R grid points [len]
-    std::vector<Turn> grid_phi;  //!< Phi grid points [AU]
-    std::vector<real_type> grid_z;  //!< Z grid points [len]
+    std::vector<float> grid_r;  //!< R grid points [len]
+    std::vector<Turn_t<float>> grid_phi;  //!< Phi grid points [AU]
+    std::vector<float> grid_z;  //!< Z grid points [len]
 
-    std::vector<real_type> field;  //!< Flattened R-Phi-Z field component
-                                   //!< [bfield]
+    std::vector<float> field;  //!< Flattened R-Phi-Z field component
+                               //!< [bfield]
```

Comment:
It's probably better to have the input be in standard `real_type` precision, so we could in theory check for catastrophic precision loss when we downconvert to double.